### ISSUE TITLE
Dataset referred in the paper

### ISSUE THREAD

**Issue by Saqhas** [MIPT-Oulu/DeepKnee#14]
Can you please share the link to the dataset created and made public by the authors of the paper as refered in the paper.

**Comment by lext** [comment]
Switch to the master branch.

**Comment by Saqhas** [comment]
Thank you. I will check

**Comment by ghost** [comment]
Can you please share the link to download so that i can download it?

**Comment by Saqhas** [comment]
@lext  I would also appreciate if you can put the link to the dataset here?

**Comment by ghost** [comment]
@Saqhas  did you get any link for data or download it?

**Comment by ghost** [comment]
I have requested to access data in the Osteoarthritis Initiative but it takes 2 months from now the website is still not working
can you please share the dataset?

**Comment by Saqhas** [comment]
@hammadyounas2008  I also don't have this dataset. Neither some link to download it.
I am also not able to get the OAI and MOST dataset as they have changed their guidelines. My application is also in the queue.
If you have the MOST dataset can you share it with me.
@lext  Can you provide the link to you dataset please?
It will be very helpful for us.

**Comment by ghost** [comment]
No, i am also looking for these datasets but dnt get any link to download

**Comment by eeshahid** [comment]
I am also looking for the dataset referred to in the paper. Or the OAI and MOST datasets.

**Comment by Saqhas** [comment]
Hi @eeshahid  you can find the OAI dataset at their project page: https://nda.nih.gov/oai/

Just select the dataset you need from the packages list and follow the procedure to download it.

Similarly MOST dataset can be found on their website: https://most.ucsf.edu/multicenter-osteoarthritis-study-most-public-data-sharing
